cmd/jsonApplyPatch_test.go

```diff
@@ -71,8 +71,8 @@ func TestSchemaPatch(t *testing.T) {
 			Output: "output.json",
 		}
 		filesMock := mock.FilesMock{}
-		filesMock.AddFile("schema.json", schema)
-		filesMock.AddFile("patch.json", patch)
+		filesMock.AddFile("schema.json", schema, 0644)
+		filesMock.AddFile("patch.json", patch, 0644)
```

Comment:
I would much prefer a second method AddFileWithPermissions(), which is called from AddFile() with the default of 0644. Having to adapt every test and specifying it again and again is very unconvenient and violates DRY.

Reply:
done